django_mailbox/models.py

```diff
@@ -96,6 +98,14 @@ class Mailbox(models.Model):
         default=True,
     )
 
+    last_polling = models.DateTimeField(
+        _(u"Last polling"),
+        help_text=(_("The time of last successfull polling of message. "
```

Comment:
I understand everything. I update code and merge in the morning. In my country there is 1:50  AM.